src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -1087,6 +1087,85 @@ <h2 class="modal-title">
         </div>
         <!-- End custom validation field -->
 
+        <!-- Short Text -->
+
+        <div
+          ng-show="vm.field.fieldType === 'textfield'"
+          class="edit-fields-line"
+        ></div>
+
+        <div ng-if="vm.field.fieldType === 'textfield'" class="row">
+          <div class="row"><br /></div>
+          <div class="toggle-option">
+            <div class="col-xs-8 label-custom label-medium">
+              Enable pre-fill
+              <i
+                class="glyphicon glyphicon-question-sign"
+                uib-tooltip="Allow this field to be pre-populated via URL"
+                tooltip-trigger="'click mouseenter'"
+              >
+              </i>
+            </div>
+            <div class="col-xs-4 field-input">
+              <label
+                class="toggle-selector pull-right"
+                ng-class="vm.field.allowPrefill ? 'toggle-selector-on' : ''"
+                onclick=""
```

Comment:
is the `onclick` property suppose to be empty?

Reply:
yes, it is to provide the mouseover hand, the actual logic comes from the checkbox being ticked

Reply:
you should use the CSS `cursor` property

Reply:
edited